commit bc60efb19ab853e27fbb1f9fb00d64a716804786 (refs/changes/34/3334/4)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-02-08T23:20:15+00:00 (1 year, 8 months ago)
    
    TRITON-118 volapi should only look for "ready" volumes when checking volumes reachability
    Reviewed by: Nick Zivkovic <zivkovic.nick@gmail.com>
    Approved by: Nick Zivkovic <zivkovic.nick@gmail.com>

diff --git a/lib/endpoints/vms.js b/lib/endpoints/vms.js
index 1c25c7e..33c09ec 100644
--- a/lib/endpoints/vms.js
+++ b/lib/endpoints/vms.js
@@ -1332,7 +1332,8 @@ function createVm(req, res, next) {
 
                 req.app.volapi.listVolumes({
                     name: volumeName,
-                    owner_uuid: ownerUuid
+                    owner_uuid: ownerUuid,
+                    state: 'ready'
                 }, function onListVols(listVolsErr, vols) {
                     var checkReachableErr;
                     var foundVolNetwork = false;
